setup.py

```diff
@@ -0,0 +1,7 @@
+from setuptools import setup
+
+setup(name='castor',
+      version='1.0.0',
+      description='models for question answering',
+      packages=['sm_model'],
+      )
```

Comment:
@salman1993 this follows the standard format: https://docs.python.org/3/distutils/setupscript.html
Can we ship it, if there's no other comment?